hatchways/team-cosmic-crisp
# Notification booking

### What this PR does (required):
- Integrate notification and booking

### Screenshots / Videos (required):

https://user-images.githubusercontent.com/6465981/121626600-97d9c280-ca2a-11eb-8ed0-b264c0900747.mp4



### Any information needed to test this feature (required):
- using two different browsers to login 2 different accounts. sitter@email.com, 123456; owner@email.com, 123456
- send a sitting request using owner, sitter should be able to see new notification after refresh.
- sitter approve owner's request, owner should be able to see 'request accept' notificaiotn
- after sitter's approve, owner can pay for the dog sitting, after payment is successful, sitter's will receive a new notification saying the payment was successful.
- click mark all read should mark all notifcaotins to read.

### Any issues with the current functionality (optional):
